src/codegen/function_body_chunk.rs

```diff
@@ -339,7 +318,7 @@ impl Builder {
         } else {
             Chunk::FromGlibConversion {
                 mode: parameter.into(),
-                array_length: parameter.array_length.clone(),
+                array_length_name: self.find_array_length_name(&parameter.name),
```

Comment:
Actually I don't, but you could fake one by editing a .gir file accordingly. The only cases that are there for real currently (or so it seems) are return-value & out-length-parameter and in-parameter & in-length-parameter